plugins/parsers/prometheus/parser_test.go

```diff
@@ -344,3 +347,37 @@ func parse(buf []byte) ([]telegraf.Metric, error) {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/vnd.google.protobuf; proto=io.prometheus.client.MetricFamily; encoding=delimited")
+		w.Write(sampleProtoBufData)
+	}))
+	defer ts.Close()
+	req, err := http.NewRequest("GET", ts.URL, nil)
+	if err != nil {
+		t.Fatalf("unable to create new request '%s': %s", ts.URL, err)
+	}
+	var resp *http.Response
+	resp, err = uClient.Do(req)
+	if err != nil {
+		t.Fatalf("error making HTTP request to %s: %s", ts.URL, err)
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		t.Fatalf("error reading body: %s", err)
+	}
+	parser := Parser{Header: resp.Header}
+	_, err = parser.Parse(body)
+	if err != nil {
+		t.Fatalf("error reading metrics for %s: %s",
+			ts.URL, err)
+	}
```

Comment:
It would be nice if we can check the returned metric:
```suggestion
	metrics, err = parser.Parse(body)
	if err != nil {
		t.Fatalf("error reading metrics for %s: %s", ts.URL, err)
	}
	testutil.RequireMetricsEqual(t, expected, metrics, testutil.SortMetrics())
```
You just need to define `expectated` accordingly (see code above).

Reply:
Ok. I will make it later. I think next Monday i can do it

Reply:
Thx!

Reply:
Done